src/Settings/components/comon/ListitemWithIcon.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { ListItem, Icon } from "@rneui/themed";
+import { useSelector } from "react-redux";
+import { iconNightColor, nightModeStyles, nightModeColor } from "../../styles/nightModeStyles";
+
+const ListItemWithIcon = ({ iconName, title, navigate, navigationTarget }) => {
+  const isNightMode = useSelector((state) => state.isNightMode);
+  const iconColor = iconNightColor(isNightMode);
+  const { containerNightStyles } = nightModeStyles(isNightMode);
+  const nightColor = nightModeColor(isNightMode);
```

Comment:
All three values here `iconColor` `containerNightStyles` `nightColor` are recomputed on every render. We can wrap them up in useMemo